gpytorch/priors/wishart_prior.py

```diff
@@ -25,7 +25,7 @@ def __init__(self, nu, K):
             raise ValueError("Must have nu > n - 1")
         super(WishartPrior, self).__init__()
         self.register_buffer("K_inv", torch.inverse(K))
-        self.register_buffer("nu", torch.Tensor([nu]))
+        self.register_buffer("nu", torch.tensor(nu))
```

Comment:
let's use `float(nu)` here, o/w we may end up seeing int/float issues in pytorch

Reply:
Fixed.